### PR TITLE
Karen feature

### PR DESCRIPTION
**Pull request by lacaine**
please kindly review/advise

### CODE REVIEW

**Comment by rnfor-pro** [comment]
awesome

File: file2 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# thiss scrfipt is to create a directory in the default user
+echo -n " please enter the directory name: "
+read directory
```
Comment (rnfor-pro) [inline file2:4]: great

**Review by rnfor-pro — APPROVED** [review]

**Review by Richard-Tass — APPROVED** [review]
Looks great.

**Comment by Richard-Tass** [comment]
Great!

**Pull request merged**